haskell/skynet.cabal

```diff
@@ -0,0 +1,28 @@
+name:                skynet
+version:             0.1.0.0
+synopsis:            Simple project template from stack
+description:         Please see README.md
+homepage:            http://github.com/bitemyapp/skynet#readme
```

Comment:
Shouldn't this be @atemerev's repo? (I'm not sure what the conventions for gitHub and cabal are.)